.github/workflows/rnmobile-ios-runner.yml

```diff
@@ -2,16 +2,18 @@ name: React Native E2E Tests (iOS)
 
 on:
     pull_request:
+        type: [labeled, opened, synchronize, reopened]
     push:
         branches: [trunk]
 
 jobs:
     test:
+        if: ${{ github.event.label.name == 'Mobile App - i.e. Android or iOS' }}
```

Comment:
Ideally we wouldn't have to duplicate the workflow or large parts of it, but not sure how to do that so for now I'm just making the test run only on mobile PRs (this is just an experiment, the plan is not to merge this change as-is).

Reply:
To clarify, the idea I have is to run only a small UI test suite on web PRs and run all UI tests on mobile PRs. So hopefully I'll find a way to do that with a conditional if-else that doesn't duplicate lots of code. For now I've made these mobile UI tests run _only_ on mobile PRs.